cylc/flow/task_proxy.py

```diff
@@ -263,10 +263,18 @@ def __repr__(self) -> str:
         return f"<{self.__class__.__name__} '{self.tokens}'>"
 
     def __str__(self) -> str:
-        """Stringify with tokens, state, submit_num, and flow_nums."""
+        return self.get_str()
+
+    def get_str(self, state=True) -> str:
+        """Stringify with tokens, state, submit_num, and flow_nums.
+
+        Args:
+            state: Include/omit state from string.
+        """
```

Comment:
This looks unused other than in `__str__()` from a quick search

Reply:
Well noticed - this is garbage left over from the earlier approach of scraping `job-activity.log`.